src/main/java/org/broadinstitute/hellbender/utils/help/HelpConstants.java

```diff
@@ -65,7 +65,10 @@ public static String forumPost(String post) {
     public final static String DOC_CAT_VARDISC = "Variant Discovery Tools";
     public final static String DOC_CAT_VAREVAL = "Variant Evaluation Tools";
     public final static String DOC_CAT_VARMANIP = "Variant Manipulation Tools";
-
+    
+    public final static String DOC_CAT_ANNOTATORS = "Annotation Modules";
+    public final static String DOC_CAT_ANNOTATORS_SUMMARY = "Annotations available to HaplotypeCaller, Mutect2 and VariantAnnotator";
```

Comment:
I wonder if we should say "Variant Callers" and "VariantAnnotator" rather than enumerating here, so we don't cause confusion by leaving out Sparkk tools (HaplotypeCallerSpark).

Reply:
I think that the annotations extend to Spark versions should be understood.

Reply:
I'd like to make these more future-proof by removing the tool names (a dev could add a new tool that annotates things at any time). Can't make it about variant callers only (VA is not a caller, and again, want to be future proof) so should try  to describe this in a more generic way.

Reply:
I support @vdauwera idea - I will use the annotators for some of my downstream tools, and this summary is tool-specific although other tools might use them...